packages/fast-components-styles-msft/src/badge/index.ts

```diff
@@ -68,6 +67,7 @@ const styles: ComponentStyles<BadgeClassNameContract, DesignSystem> = (
             display: "inline-block",
             maxWidth: "215px",
             color: neutralForegroundRest,
+            transition: "all 0.2s ease-in-out",
```

Comment:
Badge doesn't respond to interaction, so I don't think it needs transition timing.